### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by fleupold**
don't read me, read the README...

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
...
+
+```
+export PK=<private key with some funds>
+export INFURA_KEY=<infura key> // Not needed on xDAI
+```
+
+You also need a [@uniswap/token-lists](https://github.com/Uniswap/token-lists) style token list from which a random trade will be determined (e.g. the [Uniswap default list](https://raw.githubusercontent.com/Uniswap/token-lists/master/test/schema/bigexample.tokenlist.json)).
+
+```
+yarn hardhat trade --token-list-url <token list> --network <network>
```
Comment (bh2smith) [inline README.md:27]: Does the trade just make some random trade? Or is this trade deterministic? Might be worth mentioning what the trade command is about to initiate.

**Review by bh2smith — APPROVED** [review]
I like it but the run command could use some elaboration.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,36 @@
...
+```
+
+To run the trade script, you need an account (in form of a mnemonic or private key) with some funds on you target network and potentially an Infura key (unless you are running on xDAI)
+
+```
+export PK=<private key with some funds>
+export INFURA_KEY=<infura key> // Not needed on xDAI
+```
+
+You also need a [@uniswap/token-lists](https://github.com/Uniswap/token-lists) style token list from which a random trade will be determined (e.g. the [Uniswap default list](https://raw.githubusercontent.com/Uniswap/token-lists/master/test/schema/bigexample.tokenlist.json)).
```
Comment (nlordell) [inline README.md:24]: Maybe this could be the default parameter if none is set?

**Review by nlordell — APPROVED** [review]
Wonderful

File: README.md (hunk 2 above)
Comment (fleupold) [reply README.md:24]: It won't work on xDAI though :-/

Comment (nlordell) [reply README.md:24]: But we have a default xDAI token list right?

Comment (bh2smith) [reply README.md:24]: The honeyswap list could be a good default

Comment (fleupold) [reply README.md:24]: The issue is that quite a few tokens in this list actually don't have sufficient liquidity to trade via uniswap. I think I might have to add a bit more sophisticated logic when it comes to picking a random token pair.

**Comment by fleupold** [comment]
Will add a default token list for each network once #9 is merged.

**Pull request merged**